content/index.html

```diff
@@ -36,40 +36,38 @@ <h3>LXD</h3>
           <p>LXD is a next generation system container and virtual machine
             manager. It offers a unified user experience around full Linux
             systems running inside containers or virtual machines.</p>
-          
+
           <p>LXD is image based and provides images for a wide number of
             Linux distributions. It provides flexibility and scalability
             for various use cases, with support for different storage
             backends and network types and the option to install on
             hardware ranging from an individual laptop or cloud instance
             to a full server rack.</p>
-          
+
           <p>When using LXD, you can manage your instances (containers and VMs)
             with a single command line tool, or over the network in a transparent way
             through a REST API.</p>
         </div>
 
-        <div class="col-4">    
+        <div class="col-4">
           <p>
             <a class="p-button--positive" href="/lxd/try-it/" role="button">Try it online</a>
           </p><p>
             <a class="p-button" href="/lxd/introduction/" role="button">Learn more</a>
           </p>
         </div>
-        
+
       </div>
     </div>
   </div>
-      
+
   <div class="row u-equal-height">
     <div class="p-card col-4">
       <h3>LXC</h3>
       <p>LXC is a well-known Linux container runtime that consists of
         tools, templates, and library and language bindings. It's pretty
         low level, very flexible and covers just about every containment
         feature supported by the upstream kernel.</p>
-      <p>LXC is production ready with LTS releases coming with 5 years
```

Comment:
@ru-fu is this covered elsewhere now? Thanks

Reply:
Not in this exact wording, but https://linuxcontainers.org/lxc/introduction/#support is more precise on this topic.